doc/sources/patching/patching-options.rst

```diff
@@ -28,6 +28,10 @@
     from sklearnex import patch_sklearn
     patch_sklearn()
 
+- Through importing the desired estimator from the sklearnex module in your script::
+
+    from sklearnex.neighbors import NearestNeighbors
+
 - Through :ref:`global patching <global_patching>` to enable patching for your scikit-learn installation for all further runs::
 
-    python sklearnex.glob patch_sklearn
\ No newline at end of file
+    python sklearnex.glob patch_sklearn
```

Comment:
Lets' also add wording that those different ways for enabling our optimizations are interchangeable and can be combined. 
I would also add details section below for advanced users on how this functionality works technically

Reply:
something like this is already in the documentation
https://intel.github.io/scikit-learn-intelex/what-is-patching.html#patching

I will add technical information